example/src/main/java/io/singularitynet/sdk/example/CntkImageRecognition.java

```diff
@@ -0,0 +1,62 @@
+package io.singularitynet.sdk.example;
+
+import java.math.BigInteger;
+
+import io.singularitynet.sdk.client.Configuration;
+import io.singularitynet.sdk.client.JsonConfiguration;
+import io.singularitynet.sdk.client.Sdk;
+import io.singularitynet.sdk.client.PaymentStrategy;
+import io.singularitynet.sdk.client.FixedPaymentChannelPaymentStrategy;
+import io.singularitynet.sdk.client.ServiceClient;
+
+import recognition.RecognizerGrpc;
+import recognition.RecognizerGrpc.RecognizerBlockingStub;
+import recognition.ImageRecon.Input;
+import recognition.ImageRecon.Output;
+
+public class CntkImageRecognition {
+
+    public static void main(String[] args) throws Exception {
+        String privateKey = args[0];
+        BigInteger channelId = new BigInteger(args[1]);
+
+        String json = "{" +
+            "\"ethereum_json_rpc_endpoint\": \"https://ropsten.infura.io\", " +
+            "\"ipfs_url\": \"http://ipfs.singularitynet.io:80\"," +
+            "\"signer_type\": \"PRIVATE_KEY\"," +
+            "\"signer_private_key_base64\": \"" + hexToBase64(privateKey) + "\"" +
+            "}";
+        Configuration config = new JsonConfiguration(json);
+
+        Sdk sdk = new Sdk(config);
+        try {
+
+            PaymentStrategy paymentStrategy = new FixedPaymentChannelPaymentStrategy(
```

Comment:
python and js sdk currently takes paymentStartegy as sdk input parameter not as  service_client parameter , if it makes sense all three sdk should have common api.

Reply:
I agree that all three SDK should use same API. But Python and JS SDK takes payment strategy as parameter of `create_service_client` method, see links below:
Python SDK: https://github.com/singnet/snet-cli/blob/eca41c7e0549fc6bc5aef5a09c18495893fb378f/packages/sdk/snet/sdk/__init__.py#L58-L66
Node.js SDK: https://github.com/singnet/snet-sdk-js/blob/a315f7e95da0e39ac436ed11aff417d3224d70d0/packages/nodejs/src/NodeSdk.js#L14-L18
Web SDK: https://github.com/singnet/snet-sdk-js/blob/a315f7e95da0e39ac436ed11aff417d3224d70d0/packages/web/src/WebSdk.js#L13-L17

I believe it is correct solution as each may have different payment methods etc, so it may require different strategies.